net/test/demo/demo-test-run.sh

```diff
@@ -49,11 +49,11 @@ main()
 
 	read line
 	CONSOLE_SSH=$(echo "$line" | awk "{print \$1}")
-	CONSOLE_CMD="$(echo $line | cmd_get)"
+	CONSOLE_CMD="$(echo "$line" | cmd_get)"
```

Comment:
I think this can possibly be miss parsed into "$(echo "   and then $line   and then "| cmd_get)"
hence, I would suggest to leave this occurrence as it is.